ci/unit_tests/functions_test_evaluate/main_data/test_multi_junitxml_ref.eval.junit.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<testsuites>
+	<testsuite errors="6" failures="1" name="test_multi_junitxml.out" skipped="0" tests="11" time="50.0" timestamp="2019-05-17 17:16:54.437212">
+		<testcase>
+			<error message="Input test array element 0 is not dictionary. Each test has to be dictionary, please see doc!" type="ValueError"/>
+		</testcase>
+		<testcase name="Cannot open expected output">
+			<error message="Cannot open expected output payload from file: /some/random/path" type="IOError"/>
+		</testcase>
+		<testcase name="Cannot decode json from expected output">
+			<error message="Cannot decode json from expected output payload from file main_data/no.json, error: Expecting value: line 1 column 1 (char 0)" type="ValueError"/>
+		</testcase>
+		<testcase name="Cannot open returned output">
+			<error message="Cannot open returned output payload from file: /some/random/path" type="IOError"/>
```

Comment:
I would prefer unified approach, e.g. without colon and with quotes around path (and similarly for test type)
```suggestion
			<error message="Cannot open returned output payload from file '/some/random/path'" type="IOError"/>
```